.github/workflows/beta-release.yml

```diff
@@ -0,0 +1,84 @@
+name: Beta Release
```

Comment:
This looks pretty good! I'm interested in there's ways to reuse the yaml more effectively, but I think this is good for now. One thing I think we need though is a condition that this will only run on pushes to the `develop` branch. And then the stable release should only run on tagged pushes to `main`

Reply:
I think that's what this does, right? Re: only running on pushes to develop: https://github.com/washingtonpost/FastFEC/pull/14/files#diff-18c71efd7026f4953ff7ac4065357d2ea0e5e4c887cb831f706c42ea2eaa8cb2R5

Reply:
And similarly for the stable release on tagged pushes to main I believe that's what this is requiring: https://github.com/washingtonpost/FastFEC/pull/14/files#diff-17341a7895b6ea06ca7933e27af505be9c00b3034f95b7310faf905a041f8625R3-R10

Reply:
I think I'm looking for a top-level action rather than something within each job. Check this out:

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions

It should be something like:

```
on:
  push:
    branches:
      - develop
```

Reply:
And the stable release should have a branch added in the same way that only looks in `main`